art/art.py

```diff
@@ -221,12 +224,12 @@ def tsave(
         files_list = os.listdir(os.getcwd())
         extension = ".txt"
         splitted_filename = filename.split(".")
-        name = splitted_filename[0]
+        name = filename[:-1 * filename[::-1].find('.') - 1]
```

Comment:
It seems you should change this line

Reply:
You shouldn't remove `filename` without extension, for example default `filename` = art